packages/axios-extension/src/abap/abap-service-provider.ts

```diff
@@ -141,7 +130,7 @@ export class AbapServiceProvider extends ServiceProvider implements AbapServiceP
      *
      * @returns an instance of the UI5 ABAP repository service.
      */
-    public ui5AbapRepository(): Ui5AbapRepositoryService {
+    public getUi5AbapRepository(): Ui5AbapRepositoryService {
```

Comment:
Can the renaming of these methods be mentioned in the change set? These are breaking changes

Reply:
> Also, does this need updating now? 
README is no longer relevant. Deleted this README.md file.

> Could you take care of the linter warnings please? 
Fixed all the linting warnings.

Reply:
Changeset looks good. Could you please also add subsection about the transport service to https://github.com/SAP/open-ux-tools/tree/fix/711/create-transport-request/packages/axios-extension#abap-service-provider? Ideally with a code snippet since it is a little different than the other services.

Reply:
Updated README with ADT service usage. Updated the MD link syntax to the correct adoc link.